src/role-access/utils/role-access-utils.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { InfoMessageType } from 'src/work-allocation-2/enums';
```

Comment:
@connorpgpmcelroy - please change the import to a relative one instead of absolute

Reply:
yep, will do